lib/src/base-styles/_nx-tables.scss

```diff
@@ -10,141 +10,173 @@
+        box-shadow: 0 0 3px 1px transparentize(#02a6d9, 0.7);
+        outline: none;
+
+        > .nx-cell {
+          border-color: #02a6d9;
+        }
+      }
     }
+  }
+}
 
-    &:first-child .nx-cell {
-      border-top-style: none;
+.nx-table--scrollable {
+  border-style: none;
+  border-radius: 0;
+  overflow: visible;
+
+  // In Chrome (at least), tables are forced to have a whole number of pixels for their width. This means that
+  // if a container has a fractional number of pixels width, the table width will be truncated down and a visible
+  // gap will appear.  This calc ensures that when in a fractional-width container, the table will get an extra
+  // pixel of width
+  width: calc(100% + 0.9999px);
+
+  > thead > :first-child, > tbody > :last-child, {
+    > .nx-cell {
+      &:first-child, &:last-child {
+        border-radius: 0;
+      }
     }
+  }
 
-    .nx-cell--label {
-      @include semi-bold();
+  // create a stacking context so the headers don't appear over other content outside the table
+  position: relative;
+  z-index: 0;
 
-      color: $nx-text-color;
-    }
+  .nx-cell--header {
+    position: sticky;
+    top: 0;
+    z-index: 1;
   }
 }
 
-.nx-table-row--header {
-  border-bottom: $nx-border-row-dark;
+.nx-table--fixed-layout {
+  table-layout: fixed;
+
+  .nx-cell--icon {
+    box-sizing: content-box;
+
+    // the width:1px that works well with auto-layout tables is taken literally in fixed-layout tables.
+    // Instead, we make a reasonable guess at the width of the icon here.
+    width: 1em;
+
+    &:last-child {
+      // in case the icon is less than 1em wide, still put it the correct distance from the right edge
+      // (typical case: chevrons)
+      text-align: right;
+    }
+  }
 }
 
 .nx-cell {
   @include container-vertical;
   @include container-horizontal;
 
-  background-clip: padding-box;
-  border-top: $nx-border-row;
-  box-sizing: border-box;
-  color: $nx-text-color;
-  display: table-cell;
-  height: 39px; // 38px cell, 1px top border
-  padding: $nx-spacing-xs $nx-spacing-sm;
-  position: relative;
-  text-align: left;
+  padding: $nx-spacing-sm $nx-spacing-md;
+}
 
-  &.nx-error {
-    border-color: $nx-fail-color-bg;
-    border-bottom-color: $nx-fail-color;
-    color: $nx-text-color-error;
-    text-align: center;
-  }
+.nx-cell--label {
+  @include semi-bold();
 }
 
 .nx-cell--header {
-  @include semi-bold;
-  border-bottom: $nx-border-row-dark;
-  background-color: $nx-off-white;
+  @include bold;
+  background-color: #bbc1de;
   border-top: none;
+  box-sizing: border-box;
 
-  color: $nx-text-color;
-  font-size: 12px;
-  height: 24px;
-  line-height: 12px;
+  color: $nx-text-color-dark;
+  font-size: $nx-font-size-s;
+  height: 48px;
   text-align: left;
   text-transform: uppercase;
-}
-
-.nx-cell--label {
-  @include semi-bold();
-
-  color: $nx-text-color;
+  white-space: nowrap;
 }
 
 // columns whose content is numerical
 .nx-cell--num {
   text-align: center;
```

Comment:
The Zeplin files show numbers left aligned, but I thought there had been a conversation about numbers being centred. @dturner32 ?

Reply:
> The Zeplin files show numbers left aligned, but I thought there had been a conversation about numbers being centred. @dturner32 ?

yes they should be centered, I just forgot to update the mocks in zeplin. I will fix that

Reply:
> > The Zeplin files show numbers left aligned, but I thought there had been a conversation about numbers being centred. @dturner32 ?
> 
> yes they should be centered, I just forgot to update the mocks in zeplin. I will fix that

just updated zeplin